packs/tfe_fdo_nomad/templates/tfe.nomad.tpl

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+job [[ .tfe_fdo_nomad.job_name | quote ]] {
+  datacenters = [[ .tfe_fdo_nomad.datacenters  | toStringList ]]
+  namespace   = [[ .tfe_fdo_nomad.tfe_namespace | quote ]]
+  type        = "service"
+
+  group "tfe-group" {
+    count = [[ .tfe_fdo_nomad.tfe_group_count ]]
+    
+    spread {
+      attribute = "${node.unique.id}"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "60s"
+      interval = "10m"
+      mode     = "fail"
+    }
+
+    update {
+      max_parallel      = 1
+      min_healthy_time  = "30s"
+      healthy_deadline  = "12m"
```

Comment:
12 _minutes_ to become healthy? Is this container really huge or something?

Reply:
Yes, it's a generous number based on out experiences and takes a lot of time to come up, it's a container with around 15 services.

Reply:
Oof, ok.